### ISSUE TITLE
wrong conv6

### ISSUE THREAD

**Issue by foralliance** [yxlijun/Pyramidbox.pytorch#14]
在你的code中，conv6的形式是：
conv6 = nn.Conv2d(512, 1024, kernel_size=3, padding=6, dilation=6)
但这个形式其实是SSD中的。
而在PyramidBox中，底层是基于S3FD中的VGG16。跟SSD没关系。所以conv6的形式应该是：
conv6 = nn.Conv2d(512, 1024, kernel_size=3, padding=1)
